### DISCUSSION TITLE
Is there any way to exit progress immediately?

### DISCUSSION THREAD

**Discussion by Williamclouf** [IBM/mac-ibm-notifications#150] [Q&A] [answered]
Is there any way to exit itself immediately after calling a banner or alert notification?
I don't want to wait for user interaction for i just want to call a notification
I saw the `timeout` option, but i'm looking for a more elegent way

**Comment by SMartorelli** [comment, marked as answer]
Hi @Williamclouf you can do it in a couple of way:
* Kill the IBM Notifier process that you triggered, it will not affect the banner/alert;
* Add `&` at the end of the banner/alert invocation in order to spin the process to a sub-shell and be free to continue with your script, e.g. `.../IBM\ Notifier -type banner -title "This is a title" &`
